layersvt/api_dump.h

```diff
@@ -53,12 +61,22 @@
 #define API_DUMP_ENV_VAR_DETAILED_OUTPUT "VK_APIDUMP_DETAILED"
 #define API_DUMP_ENV_VAR_NO_ADDRESSES "VK_APIDUMP_NO_ADDR"
 #define API_DUMP_ENV_VAR_FLUSH_FILE "VK_APIDUMP_FLUSH"
+#define API_DUMP_ENV_VAR_OUTPUT_RANGE "VK_APIDUMP_RANGE"
 
 enum class ApiDumpFormat {
     Text,
     Html,
 };
 
+static const int FRAME_RANGE_UNLIMITED = 0;
+static const int FRAME_RANGE_INTERVAL_DEFAULT = 1;
```

Comment:
Your constants are `const int`, but your range data below is `uint64_t`.  Some compilers will throw errors at the signed/unsigned and size discrepancy.  The above should likely be uint64_t.

Reply:
Good catch, fixed that.

Reply:
Fixed